core/src/main/java/apoc/refactor/GraphRefactoring.java

```diff
@@ -452,6 +458,76 @@ public void categorize(
         }
     }
 
+    @Procedure(mode = Mode.WRITE)
+    @Description("apoc.refactor.deleteAndReconnect([pathLinkedList], [nodesToRemove], {config}) - Removes some nodes from a linked list")
+    public Stream<GraphResult> deleteAndReconnect(@Name("path") Path path, @Name("nodes") List<Node> nodesToRemove, @Name(value = "config", defaultValue = "{}") Map<String,Object> config) {
+
+        RefactorConfig refactorConfig = new RefactorConfig(config);
+
+        List<Node> nodes = IterableUtils.toList(path.nodes());
+        Set<Relationship> rels = Iterables.asSet(path.relationships());
+
+        if (!nodes.containsAll(nodesToRemove)) {
```

Comment:
should the nodes just be filtered out instead of raising an error here?
If none of the nodes are in the list, then, do nothing?
WDYT?

Reply:
Yep, I agree. Changed